apis/placement/v1beta1/binding_types.go

```diff
@@ -18,7 +18,7 @@ import (
 // +kubebuilder:printcolumn:JSONPath=`.metadata.creationTimestamp`,name="Age",type=date
 
 // ClusterResourceBinding represents a scheduling decision that binds a group of resources to a cluster.
-// It MUST have a label named `CRPTrackingLabel` that points to the cluster resource policy that creates it.
```

Comment:
could you please fix it in a separate PR? to update any API comment, we need to regenerate the charts and our API reference.